### ISSUE TITLE
Removing DRM from Nook 

### ISSUE THREAD

**Issue by wicked103** [apprenticeharper/DeDRM_tools#852]
I'm trying to remove the DRM from multiple nook books that I've already purchased. I want to move them to a new kindle e-reader I bought. I downloaded the plug-in to use with calibre, but it doesn't seem to be working. I tried customizing the DeDRM plugin by configuring for B&N Nook and then creating a new keyfile, but it isn't accepting my e-mail or password (I triple checked that they're correct). Any thoughts or help on trying to get this solved is much appreciated. Thanks!

**Comment by ElleKayEm** [comment]
The methods for getting the key with these tools stopped working when B&N cut off the old Nook for PC and Nookstudy apps.  If you have a key that worked previously, it may still work (transfer it from an old setup.)  Otherwise, it's possible to find it on a rooted Android device with the Nook app or a rooted Nook device.

**Comment by wicked103** [comment]
Well darn. Ok, could you please specify what you mean by a "rooted" nook device? Is that like jailbreaking? Trying to figure out what terms to search for so I can find a good guide. Sorry I'm a bit over my head, just desperate to get my books! And thanks so much for your help!

**Comment by ElleKayEm** [comment]
Root is complete access to the entire system.  Not to be messed with lightly.  The xda developers forum has a Nook section, but I don't know how active or up-to-date info is there these days.

**Comment by wicked103** [comment]
Thank you for the help! Do you know the file format of the key file so that once I have root access I know I’m pulling the right data?

**Comment by ElleKayEm** [comment]
It's in an SQLite database.

**Comment by ElleKayEm** [comment]
From what I can recall, the key is referred to as the ccHash.  It's 28 characters long ending with an equal sign.  Put that in a txt file, but give it the extension .b64.  Then import that into the B&N section of the DeDRM plugin customization.

**Comment by ktjones** [comment]
Looking for an update on this issue.  Is retrieval of the key not possible with the windows version installed?

**Comment by ElleKayEm** [comment]
No, the key cannot be retrieved from the Windows 8/10 Nook app.
